docs/system/system-metrics.md

```diff
@@ -1159,3 +1161,24 @@ See also the [Slab allocator](https://blogs.oracle.com/linux/post/understanding-
 <!-- prettier-ignore-end -->
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
+
+## Uptime Metric
+
+### Metric: `system.uptime`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.system.uptime -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `system.uptime` | Counter | `s` | The time the system has been running | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
can we add some language on the precision? I.e. the unit is seconds, but the counter should be double and instrumentation should measure it with the highest precision possible. 
Accuracy, however, would depend on the instrumentation/OS/etc.

Reply:
Added a note around the precision and what should be expected